tests/integration/iop/test_dag_pull_sftp.py

```diff
@@ -77,13 +77,7 @@ def test_pull_from_sftp(iop_empty_repo):
             },
             {"xml": "extracted/aca95c.xml/aca95c.xml"},
         ]
-        assert iop_empty_repo.find_all() == expected_files
-        assert sorted(iop_empty_repo.get_all_raw_filenames()) == sorted(
-            [
-                "2022-07-30T03_02_01_content.zip",
-                "2022-09-01T03_01_40_content.zip",
-                "2022-09-03T03_01_49_content.zip",
-                "2022-09-24T03_01_43_content.zip",
-                "aca95c.xml.zip",
-            ]
-        )
+
+        assert sorted(
+            iop_empty_repo.find_all(), key=lambda x: x["xml"], reverse=True
+        ) == sorted(expected_files, key=lambda x: x["xml"], reverse=True)
```

Comment:
maybe use set equality for that, it will be more efficient

Reply:
since it's a list, do you have in mind, iterate per each element, use set, and then compare?:

like: [for set(files_from_repo) in iop_empty_repo.find_all()] == [for set(expected_file) in expected_files]

Reply:
no, why? I was thinking set(iop_empty_repo.find_all(), key=lambda x: x["xml"], reverse=True) instead of sorted

Reply:
because, we can put list in the set 😔 : 
`TypeError: unhashable type: 'dict'`

Reply:
because we cannot put list in the set 😔, we will get an error:
`TypeError: unhashable type: 'dict'`

Reply:
ok, it's not a big deal :)